test/src/main/webapp/META-INF/context.xml

```diff
@@ -20,27 +20,27 @@
 		name="jdbc/ibis4test-oracle-orcldb"
 		factory="org.apache.naming.factory.BeanFactory"
 		type="oracle.jdbc.xa.client.OracleXADataSource"
-		URL="jdbc:oracle:thin:@host.docker.internal:1521:ORCLCDB"
+		URL="jdbc:oracle:thin:@${jdbc.hostname:-localhost}:1521:ORCLCDB"
```

Comment:
Hebben we deze datasource eigenlijk nog nodig of kan die weg?

Reply:
Geen idee eingelijk, al heel lang niet meer gebruikt.